cilium/cmd/preflight_identity_crd_migrate.go

```diff
@@ -122,6 +122,8 @@ func migrateIdentities() {
 
 		case err != nil:
 			scopedLog.WithError(err).Error("Cannot allocate CRD ID. This key will be allocated with a new numeric identity")
+			// To check with reviewer if we can remove badKeys as it's not used
```

Comment:
badkeys is not used now, just want to confirm if it should be removed or just use it in logger

Reply:
I think they were supposed to be printed. Can be done as a follow up

Reply:
Thanks for your input, I will have change to bad key details per-id, and remove _badkeys slices_ as it's not required anymore.